lib/modules/versioning/redhat/readme.md

```diff
@@ -0,0 +1,5 @@
+Red Hat versioning is used with container images that are maintained by Red Hat.
+
+Corresponding definitions start with the version of the main component, followed by a hyphen, and a release/build version.
```

Comment:
```suggestion
Red Hat version definitions follow this pattern:

- the version of the main component, where major is required, but minor and patch are optional
- optionally a hyphen followed by release information
- the version of Red Hat's release as an integer, optionally followed by a timestamp like: `insert example`

Examples of valid Red hat versions:

- Simple example goes here
- Example with major, minor, patch, hypen and timestamp goes here
```

How about something like this? I'd like to have some examples of what's valid versioning, and what's not.

By the way, is there some official specification that we could link to?

Reply:
Good suggestion, thanks! 👍 

The only "specification" I know and that is publicly retrievable is [here](https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/vendor/redhat/names.md#docker-v1-namespacing-and-versioning-policy). As far as I see, this is not official and also not detailed enough to be considered a specification.